lib/ecto/adapters/sql.ex

```diff
@@ -128,6 +130,13 @@ defmodule Ecto.Adapters.SQL do
       ## Migration
 
       @doc false
+      def execute_ddl(repo, %View{}=definition, opts) do
```

Comment:
The reason this is present here, is in case the query is using `^` to reference a variable outside of context, so I need to get params and pas it to `Ecto.Adapters.SQL.query!`